test/eth/TransactionManager.spec.js

```diff
@@ -272,4 +272,34 @@ describe('lifecycle hooks', () => {
     expect(openTx.isFinalized()).toBe(true);
     expect(openHandlers.confirmed).toBeCalled();
   });
+
+  test('return error message with error callback', async () => {
+    const makeListener = () =>
+      jest.fn((tx, err) => {
```

Comment:
The user can access the returned error message from their callback function argument.